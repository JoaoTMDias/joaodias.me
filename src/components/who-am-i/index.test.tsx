import React from "react";
import { render } from "@testing-library/react";
import { WhoAmI } from "./index";

describe("<WhoAmI>", () => {
	it("should render without error", () => {
		const component = render(<WhoAmI />);
		expect(component).toMatchSnapshot();
	});
});
