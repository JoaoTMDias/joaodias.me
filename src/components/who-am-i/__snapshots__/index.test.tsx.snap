// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WhoAmI> should render without error 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section
        class="layout__container layout__section"
        id="who-am-i"
      >
        <div
          class="sc-AxiKw iJTxDc layout__row"
        >
          <div
            class="image"
            id="who-am-i-image"
          >
            <div
              class="sc-AxhCb eARWVt content-spinner isCenter"
              color="hsl(0, 0%, 60%)"
              data-testid="component-content-spinner"
              size="24"
              tabindex="-1"
            >
              <svg
                class="spinner__container"
                data-testid="component-content-svg"
                height="24"
                viewBox="0 0 50 50"
                width="24"
              >
                <circle
                  class="spinner__icon"
                  color="hsl(0, 0%, 60%)"
                  cx="25"
                  cy="25"
                  fill="none"
                  r="20"
                  stroke-width="5"
                />
              </svg>
            </div>
          </div>
          <div
            class="text"
            id="who-am-i-text"
          >
            <h3
              aria-label="Section Title: About Me. Description: I make stuff for the web."
              class="title"
            >
              I make stuff for the web.
            </h3>
            <h4
              class="lead"
            >
              I’m João — an interface designer and frontend developer based in Coimbra, Portugal.
            </h4>
            <p>
              I’ve been working in this industry for over 9 years and now mostly specialized in UX Design and Web Development for all kinds of screen sizes and users.
            </p>
            <p>
              Feedzai is where I work since September 2019, and it is one of the most proeminent software companies in Portugal.
            </p>
          </div>
        </div>
      </section>
    </div>
  </body>,
  "container": <div>
    <section
      class="layout__container layout__section"
      id="who-am-i"
    >
      <div
        class="sc-AxiKw iJTxDc layout__row"
      >
        <div
          class="image"
          id="who-am-i-image"
        >
          <div
            class="sc-AxhCb eARWVt content-spinner isCenter"
            color="hsl(0, 0%, 60%)"
            data-testid="component-content-spinner"
            size="24"
            tabindex="-1"
          >
            <svg
              class="spinner__container"
              data-testid="component-content-svg"
              height="24"
              viewBox="0 0 50 50"
              width="24"
            >
              <circle
                class="spinner__icon"
                color="hsl(0, 0%, 60%)"
                cx="25"
                cy="25"
                fill="none"
                r="20"
                stroke-width="5"
              />
            </svg>
          </div>
        </div>
        <div
          class="text"
          id="who-am-i-text"
        >
          <h3
            aria-label="Section Title: About Me. Description: I make stuff for the web."
            class="title"
          >
            I make stuff for the web.
          </h3>
          <h4
            class="lead"
          >
            I’m João — an interface designer and frontend developer based in Coimbra, Portugal.
          </h4>
          <p>
            I’ve been working in this industry for over 9 years and now mostly specialized in UX Design and Web Development for all kinds of screen sizes and users.
          </p>
          <p>
            Feedzai is where I work since September 2019, and it is one of the most proeminent software companies in Portugal.
          </p>
        </div>
      </div>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
